crates/autopilot/src/run_loop.rs

```diff
@@ -268,6 +275,16 @@ impl RunLoop {
                 return;
             }
 
+            tracing::info!("saving fee policies");
+            if let Err(err) = self
+                .persistence
+                .store_fee_policies(auction_id, fee_policies)
+                .await
+            {
+                Metrics::fee_policies_store_error();
+                tracing::warn!(?err, "failed to save fee policies");
+            }
```

Comment:
Decided to store them separately from the `Competition` in order to properly use metrics since this is not a critical part of the flow.